content/articles/writing-guide.markdown

```diff
@@ -0,0 +1,85 @@
+---
+title: Writing Guidelines
+excerpt: Guidelines & best practices for writing support docs 
+categories:
+- Support
```

Comment:
We should probably remove this caregory, as we currently don't have it.
https://support.dnsimple.com/